demos/snackbar.html

```diff
@@ -271,7 +270,7 @@ <h2 class="mdc-typography--title">Basic Example</h2>
           document.body.classList.contains('mdc-theme--dark') ? document.body.classList.remove('mdc-theme--dark') : document.body.classList.add('mdc-theme--dark');
         });
 
-        textFields.forEach(function(tf) {
+        [].slice.call(demoRoot.querySelectorAll('.mdc-textfield')).forEach(function(tf) {
```

Comment:
I think this was mis-copied from one of the fixes in the other demo file rather than from the removed L216; `demoRoot` doesn't exist here.  Should probably just use `document` in this case.